packages/expo/src/StoreReview.ts

```diff
@@ -1,21 +1,28 @@
 import { NativeModules, Platform, Linking } from 'react-native';
 import { Constants } from 'expo-constants';
 
+const {
+  ExponentStoreReview = {
+    get name() {
+      return 'ExponentStoreReview';
+    },
+  },
+} = NativeModules;
 /*
     * Platform must be iOS
     * iOS 10.3 or greater
     * `SKStoreReviewController` class is available
 */
 export function isSupported() {
-  return Platform.OS === 'ios' && NativeModules.ExponentStoreReview.isSupported;
+  return ExponentStoreReview.isSupported && ExponentStoreReview.requestReview;
```

Comment:
Rather than testing if `requestReview` is defined, we should fix `isSupported` so that it actually indicates support.